<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>12306定票软件</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>出发地：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbFrom">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>目的地：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbTo">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>出发日期：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateStart">
        <property name="date">
         <date>
          <year>2015</year>
          <month>10</month>
          <day>15</day>
         </date>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnQuery">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="mainsplitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTableWidget" name="tableTickets">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>7</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <property name="columnCount">
        <number>16</number>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>56</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>车次</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>出发站</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>到达站</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>出发时间</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>达到时间</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>历时</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>商务座</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>特等座</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>一等座</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>二等座</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>高级软卧</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>软卧</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>硬卧</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>软座</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>硬座</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>无座</string>
        </property>
       </column>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>3</verstretch>
        </sizepolicy>
       </property>
       <property name="undoRedoEnabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="action_login">
   <property name="text">
    <string>用户登录</string>
   </property>
  </action>
  <action name="action_tickets">
   <property name="text">
    <string>定时抢票</string>
   </property>
  </action>
  <action name="action_fresh_tickets">
   <property name="text">
    <string>自动刷票</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
